t/plugin/ai-request-rewrite2.t

```diff
@@ -188,19 +188,16 @@ override.endpoint is required for openai-compatible provider
                  [[{
                     "uri": "/anything",
                     "plugins": {
-                        "ai-proxy": {
+                        "ai-request-rewrite": {
```

Comment:
It seems that this test is useless, as it even got the plugin wrong and still passed. We need to add request validation for this route to ensure `override` works, making this test case a bit more meaningful.